luigi/contrib/hadoop.py

```diff
@@ -582,7 +587,15 @@ class DefaultHadoopJobRunner(HadoopJobRunner):
     def __init__(self):
         config = configuration.get_config()
         streaming_jar = config.get('hadoop', 'streaming-jar')
-        super(DefaultHadoopJobRunner, self).__init__(streaming_jar=streaming_jar)
+        cmdenvs = config.get('hadoop', 'cmdenvs', default=None)
+        archives = config.get('hadoop', 'archives', default=None)
```

Comment:
My personal opinion: Archives options was already added as part of  https://github.com/spotify/luigi/pull/1649/files , I don't think we should force people to use the configuration file. Users can simple extended this.  `HadoopJobRunner` or `DefaultHadoopJobRunner` to get this done. In particular users already doing this programmatically will have the code broken  by this (my case).